README.md

```diff
@@ -1 +1,81 @@
 # git-keeper
+
+## Purpose
+This utility is ment to backup git repositories.
+Repositories are clonned in `--mirror` mode, `tar`-ed and `gpg` encrypted, then uploaded to S3 bucket
+
+## Simple example of using docker image
+```
+cat repolist | docker run --rm -i -v /home/user/github/git-keeper/config:/config:z \
+    -e GIT_SSL_NO_VERIFY=true \
+    quay.io/app-sre/git-keeper:latest \
+    --config /config/gk_tomls \
+    --gpgs /config/gpg \
+    --subfolders backups/weeklytest1,backups/weeklytest2,backups/daily
+```
+
```

Comment:
perhaps we could talk a bit about the structure of `repolist` file (just one repo per line)